.github/workflows/test.yaml

```diff
@@ -16,3 +20,34 @@ jobs:
       - uses: actions/checkout@v2
       - name: Run ${{ matrix.command }}
         run: make ${{ matrix.command }}
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ${{ env.COMPONENT_NAME }}
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Compile component
+        run: |
+          GO111MODULE="on" go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+          cd ${{ env.COMPONENT_NAME }}
+          make compile
+      - name: Determining tests to run
+        id: determine-tests
+        run: |
```

Comment:
This is a complexity that might not be worth it. As a component author, I do now which tests I have implemented and thus hardwiring things is less fiddly.

Reply:
The reason I went with this approach is that
1) I want to see in GH Actions log that there were steps left out. If the steps would show up as "Done" but it was a hidden noop behind, then that makes it not really visible.
2) The github workflows might later be managed by modulesync and thus be the same file for all components. I'm aware that this is not the case at the moment, but I wanted to try it out PoC style, as that would also make it at least easier to copy-paste between components without having to touch the whole file